cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -264,17 +267,23 @@ createLoggingLayer ver nodeConfig' p = do
        , llConfiguration = logConfig
        , llAddBackend = Switchboard.addExternalBackend switchBoard
        , llSwitchboard = switchBoard
-       , llEKGDirect = mbEkgDirect
+       , llEKGDirect = ekgDirect
        }
 
-   startCapturingMetrics :: Trace IO Text -> IO ()
-   startCapturingMetrics tr = do
+   startCapturingMetrics :: TraceOptions
+    -> Trace IO Text
+    -> IO ()
+   startCapturingMetrics (TraceDispatcher _) _tr = do
+      pure ()
```

Comment:
Should we output that we aren't capturing metrics here?

Reply:
The metrics are still written to EKG. It is just that new logging is not doing it as this place.